portafly/doc/ux_specs/audience/accounts/Listing.md

```diff
@@ -0,0 +1,56 @@
+# Audience > Accounts > Listing
+
+* [Mockups](https://marvelapp.com/55343de/screen/67190788)
+* [Mockups JIRA ticket](https://issues.redhat.com/browse/THREESCALE-4725)
+* [Parent JIRA ticket](https://issues.redhat.com/browse/THREESCALE-4724)
+
+##### Page header
+* Create button
+  * Icon is `plus-circle`
+* Export button
+  * Icon is `pficon-save`
+
+##### Toolbar ([specs](../../global_components/toolbar.md))
+* Bulk actions:
+  * Send email
+  * Change account
+  * Change state
+* Filters:
+  * Group/org > text input ([PF specs](https://www.patternfly.org/v4/documentation/react/components/inputgroup#with-dropdown))
+  * Admin > text input ([PF specs](https://www.patternfly.org/v4/documentation/react/components/inputgroup#with-dropdown))
+  * State > Dropdown menu with multi-select checklist ([PF specs](https://www.patternfly.org/v4/documentation/react/components/select#checkbox-input))
+     * "Approved"
+     * "Pending"
+     * "Rejected"
+     * "Suspended"
+  <!-- * Plan > Dropdown menu with multi-select checklist ([PF specs](https://www.patternfly.org/v4/documentation/react/components/select#checkbox-input)) -- with filter when many options are available ([PF specs](https://www.patternfly.org/v4/documentation/react/components/select#grouped-checkbox-input-with-filtering))
+     * list of current plans -->
+
+##### Table ([PF specs](https://www.patternfly.org/v4/documentation/react/components/table))
+* Table header:
+  * Group/organization [sortable]
+  * Admin [sortable]
+  * Signup date [sortable]
+  * Applications [sortable]
+  * State [sortable]
+  * Actions
+* Table rows include a checkbox
+* Action column values:
+  * Approve
+    * Icon is `check`
+  * Activate
+    * Icon is `play-circle`
+  * [MASTER] Act as
+    * Icon is `bolt`
+
+##### Modals
```

Comment:
And also here there should be a third "Change plan" modal, no?

Reply:
@josemigallas  same as https://github.com/3scale/porta/pull/1828#discussion_r425836508